lib/SearchAndSort/components/ExpandFilterPaneButton/ExpandFilterPaneButton.js

```diff
@@ -8,35 +8,29 @@ import { FormattedMessage } from 'react-intl';
 import { PaneHeaderIconButton, Tooltip } from '@folio/stripes-components';
 
 const CollapseFilterPaneButton = ({ onClick, filterCount }) => (
-  <FormattedMessage
-    id="stripes-smart-components.numberOfFilters"
-    values={{ count: filterCount }}
+  <Tooltip
+    text={<FormattedMessage id="stripes-smart-components.showSearchPane" />}
+    sub={!!filterCount && (
```

Comment:
so `sub` might be `false`? it might create console warning of wrong type

Reply:
Thanks, fixed